docs-react-native/react-native/docs/ios/remote-notification-support.md

```diff
@@ -5,41 +5,21 @@ next: /
 previous: /react-native/docs/ios/permissions
 ---
 
-It's possible to display a notification with Notifee features from outside the app using remote notifications (a.k.a push notifications) in two ways:
- - using APNs keys
- - using `notifee_options` with our notification service extension helper
+> It is recommended to only use a notification service extension if you require an image or need to modify the contents of the notification before its displayed.
 
-It is recommended to only use a notification service extension if you require an image or need to modify the contents of the notification before its displayed.
+It's possible to display a notification with Notifee features from outside the app using remote notifications (a.k.a push notifications) by using `notifee_options` in APNs keys with our notification service extension helper.
 
-### Using APNs keys
+Adding a custom key `notifee_options` in the message payload enables Notifee to modify the notification before it is finally displayed to the end user. You can also change it as you please during this time.
 
-Notification messages sent through APNs follow the APNs payload format which allows us to be able to specify a category or a custom sound with no extra configuration on the client:
+> ⭐️ In order to bring the iOS experience closer to the Android one, you can still send **data-only/local/background** notifications for Android while using **remote/alert** notifications for iOS [by configuring the right payload.](#configure-the-payload)
```

Comment:
The terminology used is very important because the use cases are so critical. Rather than attempt to use a bunch of synonyms that are never fully defined, I want to use just the exact same words used to name the actual keys we are talking about in the JSON

```suggestion
> ⭐️ In order to bring the iOS experience - where data-only messages are rarely delivered -  closer to the Android one where data-only messages have high delivery rates, you may still send **data-only** notifications for Android while using messages with a **notification** payload for iOS [by configuring the right payload.](#configure-the-payload)
```